test/unit/Cover/helpers.js

```diff
@@ -50,11 +49,11 @@ async function assertCoverFields(
   const segment = await cover.coverSegments(coverId, segmentId);
   assert.equal(storedCoverData.productId, productId);
   assert.equal(storedCoverData.coverAsset, coverAsset);
-  bnEqual(storedCoverData.amountPaidOut, amountPaidOut);
+  expect(storedCoverData.amountPaidOut.toString()).to.be.equal(amountPaidOut);
```

Comment:
you shouldn't need that toString() here, i expect

correct me if i'm wrong

Reply:
I was throwing an error, I updated the code so `toString()` is not longer needed